server/src/main/java/org/eclipse/openvsx/publish/PublishExtensionVersionHandler.java

```diff
@@ -99,6 +111,7 @@ private ExtensionVersion createExtensionVersion(ExtensionProcessor processor, Us
             extension.setName(extensionName);
             extension.setNamespace(namespace);
             extension.setPublishedDate(extVersion.getTimestamp());
+            extension.setLastUpdatedDate(extVersion.getTimestamp());
```

Comment:
`setLastUpdatedDate` must be called on every publish, not just when the extension is created.
Please remove this statement.